charts/VariableData.ts

```diff
@@ -104,6 +108,16 @@ export class Variable {
         }
     }
 
+    @computed get years(): number[] {
+        if (this.display.yearIsDay) {
+            const epoch = moment.utc(EPOCH_DATE)
```

Comment:
Maybe we could move this to Utils and then easily add a few test cases?

Reply:
@breck7 Did you mean something like this? (I just pushed it)

```ts
export function diffDateISOStringInDays(a: string, b: string): number {
    return moment.utc(a).diff(moment.utc(b), "days")
}
```

Reply:
Ah the test cases you added like this one:

     expect(formatDay(0)).toEqual("Jan 1, 1970")

Was what I was looking for. Easy to read that and see what is going on. Thanks.